index.html

```diff
@@ -144,165 +254,227 @@ <h2 id="examples">Examples</h2>
-  <code>fetch()</code> API. That would enable them both to upgrade and downgrade those resource's "importance".
-</p>
+    <p>Developers need a way to provide the above signals for resources that are fetched through JavaScript, e.g., using the
+      <code>fetch()</code> API. That would enable them both to upgrade and downgrade those resource's "importance".
+    </p>
 
-<h3 id="providetheabilitytoreprioritizearesourceinflight">Provide the ability to re-prioritize a resource in-flight</h3>
+    <h3 id="providetheabilitytoreprioritizearesourceinflight">Provide the ability to re-prioritize a resource in-flight</h3>
 
-<ul>
-  <li>"Resource priority" is not always the right way of looking at it. For resources that are parsed on-the-fly (most notably
-    HTML and progressive images), their first buffer is often more important than their last. Developers can use the ability
-    to reprioritize resources to reflect this notion when downloading such resources.</li>
+    <ul>
+      <li>"Resource priority" is not always the right way of looking at it. For resources that are parsed on-the-fly (most notably
+        HTML and progressive images), their first buffer is often more important than their last. Developers can use the
+        ability to reprioritize resources to reflect this notion when downloading such resources.</li>
 
-  <li>There are also cases where the priority of a resource changes due to user action or condition changes. One example is the
-    loading of images, where in-viewport images (or soon-to-be in-viewport images) are of higher priority than images that
-    are further away from the viewport and therefore less likely to be seen by the user.</li>
-</ul>
+      <li>There are also cases where the priority of a resource changes due to user action or condition changes. One example
+        is the loading of images, where in-viewport images (or soon-to-be in-viewport images) are of higher priority than
+        images that are further away from the viewport and therefore less likely to be seen by the user.</li>
+    </ul>
 
-<h3 id="downgradepriorityofaniframeanditssubresources">Downgrade priority of an iframe and its subresources</h3>
+    <h3 id="downgradepriorityofaniframeanditssubresources">Downgrade priority of an iframe and its subresources</h3>
 
-<p>When developers load a third party iframe, they may wish to make sure that it does not contend on bandwidth and/or CPU with
-  the more important first party content of the page. Alternatively, they may wish to signal the browser that a certain third
-  party iframe is as important as the main page content and should be given CPU and bandwidth resources accordingly.
-</p>
+    <p>When developers load a third party iframe, they may wish to make sure that it does not contend on bandwidth and/or CPU
+      with the more important first party content of the page. Alternatively, they may wish to signal the browser that a
```

Comment:
This predates this PR, but consider "signal _to_ the browser..."

Reply:
Thanks for catching! Fixing in the next commits.